src/_guides/language/language-tour.md

```diff
@@ -158,38 +162,48 @@ mind:
-| abstract{{bii}}   | continue          | false             | new               | this              |
-| as{{bii}}         | default           | final             | null              | throw             |
-| assert            | deferred{{bii}}   | finally           | operator{{bii}}   | true              |
-| async{{lrw}}      | do                | for               | part{{bii}}       | try               |
-| async*{{lrw}}     | dynamic{{bii}}    | get{{bii}}        | rethrow           | typedef{{bii}}    |
-| await{{lrw}}      | else              | if                | return            | var               |
-| break             | enum              | implements{{bii}} | set{{bii}}        | void              |
-| case              | export{{bii}}     | import{{bii}}     | static{{bii}}     | while             |
-| catch             | external{{bii}}   | in                | super             | with              |
-| class             | extends           | is                | switch            | yield{{lrw}}      |
-| const             | factory{{bii}}    | library{{bii}}    | sync*{{lrw}}      | yield*{{lrw}}     |
+{% assign lrw = '&nbsp;<sup title="limited reserved word for asynchrony support" alt="limited reserved word for asynchrony support">2</sup>' %}
+{% assign lrw2 = '&nbsp;<sup title="limited reserved word" alt="limited reserved word">3</sup>' %}
+
+| abstract{{bii}}   | deferred{{bii}}   | if                | super             |
+| as{{bii}}         | do                | implements{{bii}} | switch            |
+| assert            | dynamic{{bii}}    | import{{bii}}     | sync*{{lrw}}      |
+| async{{lrw}}      | else              | in                | this              |
+| async*{{lrw}}     | enum              | is                | throw             |
+| await{{lrw}}      | export{{bii}}     | library{{bii}}    | true              |
+| break             | external{{bii}}   | new               | try               |
+| case              | extends           | null              | typedef{{bii}}    |
+| catch             | factory{{bii}}    | operator{{bii}}   | var               |
+| class             | false             | part{{bii}}       | void              |
+| const             | final             | rethrow           | while             |
+| continue          | finally           | return            | with              |
+| covariant{{lrw2}}          | for               | set{{bii}}        | yield{{lrw}}      |
+| default           | get{{bii}}        | static{{bii}}     | yield*{{lrw}}     |
 {:.table .table-striped .nowrap}
 
 <sup>1</sup> Words with the superscript **1**
-are *built-in identifiers*. Avoid using
+are **built-in identifiers**. Avoid using
 built-in identifiers as identifiers, and never use
 them for class or type names. Built-in identifiers exist to ease
 porting from JavaScript to Dart. For example,
 if some JavaScript code has a variable named `factory`, you don't have
 to rename it when you port the code to Dart.
```

Comment:
We could make an adjustment here: 'Avoid using built-in identifiers as identifiers. It is a compile-time error to use them for class or type names, and it is a source of confusion if they are used as names of variables or functions.'

The point is that the hard fact is stated as such, and the broader style rule about not using these names to declare other things is motivated. This will match the actual experience of developers who do it anyway.